docs/docs/conceptual/gatsby-for-ecommerce.md

```diff
@@ -0,0 +1,45 @@
+
+### Choosing additional content systems
+
+Once an organization selects a main e-commerce platform, it may also want or need other content stores that will get pulled into the website. They might choose a CMS like Contentful for complex content modelling, or Wordpress for blog content authoring. They might choose Yotpo for customer reviews or Salsify to store product SKUs. 
+
+### Why organizations build e-commerce sites with Gatsby
+
+Organizations looking for a JAMStack e-commerce site that go with Gatsby typically do so for a combination of three reasons: they want to embrace a modern development framework (React), they want to optimize performance, and they want a web framework that’s already integrated with the systems they’re using (via Gatsby’s plugin system and pre-built integrations). 
+
+From a conversion perspective, Gatsby’s lightning-fast performance is a huge win: Gatsby automates code splitting, image optimization, inlining critical styles, lazy-loading, prefetching resources, and more to ensure your site is fully optimized. 
+
+And with Gatsby’s pre-built integrations, it can pull data in from all of these sources (Shopify, plus Wordpress, Contentful, Yotpo, etc) and make it available for the React components. 
+
+### Specific e-commerce development considerations
+
+E-commerce tends to have a number of specific requirements. When building a Gatsby site (or other decoupled/JAMStack site, for that matter) sourced from an ecommerce backend like Shopify, developers will typically have to think through a few additional touchpoints between the systems:
+
+- **Persisting a cart across site pages and between sessions** (ie, if the user closes their browser and reopens it tomorrow, the items should still be there). This can handled either through local-storage or through the shopify-buy JS library.
+- **Product search** can be done client-side if the SKU count is small enough to store all products in a global state. Alternatively, it can be handled through the e-commerce provider’s search features, or if those aren’t robust enough, a third-party search provider like Algolia.
+- **Surfacing price adjustments** like tax, shipping, discounts/promos to the user while browsing the site. Different e-commerce solutions provide different levels of API access to these objects. 
+- **Handling checkout.** In order to comply with PCI regulations around storing credit card information, e-commerce providers typically exert strong control over the "buy" or "checkout" experience. Shopify requires all checkout flows using their platform to use their hosted checkout pages, though it's common to wrap them in a subdomain of the main site (eg the Gatsby/Shopify site [strivectin.com](strivectin.com] wraps a `myshopify.com` URL under a `shop.strivectin.com` URL for checkout). 
+- **Handling account pages.** Again, many sites choose to wrap their e-commerce provider’s account pages under their own domain. 
+
+Additional resources:
```

Comment:
Perhaps https://www.bigcommerce.com/articles/headless-commerce/ - as you mention BC earlier and BC specifically as to it's strategy to be differentiated from Shopify is pushing headless support hard. Toby, Shopify's founder, isn't really a convert to headless from what I have seen.

Reply:
Hadn't seen that piece, thanks! Really great overview and frames the relationship between an e-commerce platform and Gatsby well.